### PR TITLE
4.9

### PR DESCRIPTION
**Pull request by priyanka-TL**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://937.qa.docs.sunbird.org/4.9.0/ .<br/> **Note:** Only _4.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://937.qa.docs.sunbird.org/4.9.0/apis/mlcoreapi// <br /> - http://937.qa.docs.sunbird.org/4.9.0/apis/mlprojectsapi// <br />  <br/> **Modified sections are -** <br />

File: apis/mlcoreapi/mlcoreapi.yaml [hunk 1]
```diff
@@ -555,7 +555,64 @@ paths:
       deprecated: false
       security:
         - bearer: []
-
+  /solutions/mlcore/v1/read/{solutionId}:
+    get:
+      summary: Read Solution Report Informations For Given Solution Id
+      description: |-
+        The API is used to get the solution information's for the given solutionId to be broadly used in the report dashboard by program managers and program designers.
```
Comment (anisha147) [inline apis/mlcoreapi/mlcoreapi.yaml:562]: Please check the sentence formation.

File: apis/mlcoreapi/mlcoreapi.yaml [hunk 2]
```diff
@@ -555,7 +555,64 @@ paths:
...
+            To make use of the API, you require authorization. Raise a request
+            to the administrator for the use of the API. You will receive the
+            authorization key. Specify the key received, here.
+          required: true
+          schema:
+            type: string
+        - in: path
+          name: solutionId
+          required: true
+          description: Please append a valid solution id To the Request URL
```
Comment (anisha147) [inline apis/mlcoreapi/mlcoreapi.yaml:589]: lowercase To

File: apis/mlcoreapi/mlcoreapi.yaml [hunk 3]
```diff
@@ -1433,35 +1492,35 @@ components:
...
           type: string
           example: course
         programExternalId: 
-          description: 'Represent the program external id for solution'
+          description: 'Represents the program external id for solution'
           type: string
           example: TN-Program-1597301830708
         isReusable:
-          description: 'Represent the isReusable'
+          description: 'Represents the isReusable'
```
Comment (anisha147) [inline apis/mlcoreapi/mlcoreapi.yaml:1523]: Please check whether the description is complete

**Comment by amorphous-1** [comment]
Hello @priyanka-TL,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://937.qa.docs.sunbird.org/4.9.0/ .<br/> **Note:** Only _4.9.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://937.qa.docs.sunbird.org/4.9.0/apis/mlcoreapi// <br /> - http://937.qa.docs.sunbird.org/4.9.0/apis/mlprojectsapi// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**